lib/probe_dock_rspec/meta_parser.rb

```diff
@@ -46,24 +45,29 @@ def self.probedock_meta holder
       end
     end
 
-    def self.extract_name_parts example, groups = []
+    def self.extract_name_parts(example, groups = [])
       (groups.collect(&:description) << example.description).compact.collect(&:strip).reject{ |p| p.empty? }
     end
 
-    def self.extract_category example, groups = []
+    def self.extract_category(example, groups = [])
       cat = (groups.collect{ |g| probedock_meta(g)[:category] } << probedock_meta(example)[:category]).compact.last
       cat ? cat.to_s : nil
     end
 
-    def self.extract_tags example, groups = []
+    def self.extract_active(example, groups = [])
+      active = (groups.collect{ |g| probedock_meta(g)[:active] } << probedock_meta(example)[:active]).compact.last
+      active.nil? ? nil : active
```

Comment:
This line does nothing. Remove it along with the `active = ` assignment on the previous line.